src/System.Runtime.Extensions/tests/System/IO/PathTests.netcoreapp.cs

```diff
@@ -218,5 +218,19 @@ public void GetFullPath_Wildcards(char wildcard)
             string path = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName() + wildcard + "ing");
             Assert.Equal(path, Path.GetFullPath(path));
         }
-    }
+
+		[Fact]
+		public static void GetFullPath_BasePath_InvalidInput_Unix()
+		{
+			AssertExtensions.Throws<ArgumentNullException>("basePath", () => Path.GetFullPath("", null));
```

Comment:
In general you should try to use theories where you have multiple sets of input like this. Make one method for ArgumentException and one for ArgumentNull perhaps. Assert.Throws<> allows you to specify the param (which can be specified as null), you don't need to use AssertExtensions.

Additionally, where are the Windows tests for the exceptions?